### PR TITLE
EDU-352 Wrap CDN props in public storage object

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-352

### CODE REVIEW

File: src/ui/default-prop-types.js [hunk 1]
```diff
@@ -283,3 +282,9 @@ export const lessonMetadataShape = PropTypes.shape({
 export const lessonShape = PropTypes.shape({
   ...lessonMetadataProps
 });
+
+export const storageShape = PropTypes.shape({
```
Comment (ahelmberger) [inline src/ui/default-prop-types.js:286]: I think we have to be more specific with the naming to indicate that this is the config for the file picker. We will also have a storage shape on the user (the `storage` property) and a storage plan shape.

Comment (adrianaluchian) [reply src/ui/default-prop-types.js]: Indeed, renamed to `filePickerStorageShape`, thanks!

File: src/components/sections-display.js [hunk 2]
```diff
@@ -3,14 +3,14 @@ import { Button, Divider } from 'antd';
 import { PlusOutlined } from '@ant-design/icons';
 import React, { Fragment, useState } from 'react';
 import SectionDisplay from './section-display.js';
-import { sectionShape } from '../ui/default-prop-types.js';
 import PluginSelectorDialog from './plugin-selector-dialog.js';
+import { sectionShape, filePickerStorageShape } from '../ui/default-prop-types.js';
```
Comment (ahelmberger) [inline src/components/sections-display.js:7]: This is kinda "unorderly" :wink:

File: src/plugins/ear-training/editing/ear-training-sound-editor.js [hunk 3]
```diff
@@ -4,14 +4,15 @@ import { Input, Radio } from 'antd';
 import { SOUND_TYPE } from '../constants.js';
 import { useTranslation } from 'react-i18next';
 import ClientConfig from '../../../bootstrap/client-config.js';
+import { filePickerStorageShape } from '../../../ui/default-prop-types.js';
```
Comment (ahelmberger) [inline src/plugins/ear-training/editing/ear-training-sound-editor.js:7]: This too

**Review by ahelmberger — APPROVED** [review]

File: src/components/sections-display.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/sections-display.js]: 😱 Na das kann nicht sein! Korregiert!

Comment (ahelmberger) [reply src/components/sections-display.js]: Thanks :sweat_smile:

**Pull request merged**